pkg/api/convertertoagentpoolonlyapi.go

```diff
@@ -349,6 +354,20 @@ func convertV20180331AgentPoolOnlyWindowsProfile(obj *v20180331.WindowsProfile)
 	}
 }
 
+func convertV20180331AgentPoolOnlyKubernetesConfig(enableRBAC *bool) *KubernetesConfig {
+	if enableRBAC == nil || *enableRBAC == true {
```

Comment:
If `if enableRBAC == nil` shouldn't `EnableRbac` be false?

Reply:
Oh, we want the default behavior to be true.

Default behavior is when `enableRBAC == nil`

Reply:
Ok, cool. Thanks!